package.json

```diff
@@ -23,17 +23,20 @@
   "bugs": {
     "url": "https://github.com/Visual-Regression-Tracker/agent-playwright/issues"
   },
+  "engines": {
+    "node": ">=16"
+  },
   "homepage": "https://github.com/Visual-Regression-Tracker/agent-playwright#readme",
   "dependencies": {
-    "@visual-regression-tracker/sdk-js": "5.5.0"
+    "@visual-regression-tracker/sdk-js": "5.6.0"
   },
   "devDependencies": {
-    "@playwright/test": "^1.20.1",
-    "@types/jest": "^27.4.1",
-    "@types/node": "^17.0.23",
-    "jest": "^27.5.1",
-    "semantic-release": "^19.0.2",
-    "ts-jest": "^27.1.4",
-    "typescript": "^4.6.3"
+    "@playwright/test": "^1.36.0",
+    "@types/jest": "^29.5.3",
+    "@types/node": "^16.18.38",
+    "jest": "^29.6.1",
+    "semantic-release": "^21.0.7",
```

Comment:
`semantic-release` require node 18 
that was issue with sdk-js release 
we might need some pre-release step to test it for future https://github.com/Visual-Regression-Tracker/Visual-Regression-Tracker/issues/407